python/dgl/distributed/kvstore.py

```diff
@@ -0,0 +1,1107 @@
+"""Define distributed kvstore"""
+
+import os
+import time
+import random
+import numpy as np
+
+from . import rpc
+from .constants import get_type_str
+
+from .. import backend as F
+from .._ffi.ndarray import empty_shared_mem
+
+KVSTORE_PULL = 901231
```

Comment:
what is this?

Reply:
`901231` is just a unique msg_id for specified msg type.